pkg/virt-controller/watch/migration.go

```diff
@@ -448,12 +454,20 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 		log.Log.Object(migration).Error("vmi's migration state was taken over by another migration object")
 	} else if vmi.Status.MigrationState != nil &&
 		vmi.Status.MigrationState.MigrationUID == migration.UID &&
-		vmi.Status.MigrationState.Failed {
+		(vmi.Status.MigrationState.Failed || vmi.Status.MigrationState.AbortRequested) {
 
 		migrationCopy.Status.Phase = virtv1.MigrationFailed
-		c.recorder.Eventf(migration, k8sv1.EventTypeWarning, FailedMigrationReason, "Source node reported migration failed")
+		c.unmarkMigrationAborted(migration.UID)
```

Comment:
I wouldn't remove it here just because we didn't set the `MigrationFailed` status yet. This should be removed either when the object is Finalized or when we set the `MigrationFailed` state.